R/post.R

```diff
@@ -110,7 +94,7 @@ post_tweet <- function(status = "my first rtweet #rstats",
 
     ## send request
     r <- TWIT(get = FALSE, url, token)
-
+    httr::stop_for_status(r)
```

Comment:
I'd suggest leaving these off, so we can tackle in one swoop in `TWIT_get()`.

Reply:
Removed